README.md

```diff
@@ -36,9 +35,9 @@ For more information on submodules read [this chapter](http://git-scm.com/book/e
 ## Using Vipr with this Writer
 
 1. Build the solution in Visual Studio.
-2. Go to the `src\T4TemplateWriter\bin\debug` folder to find all compiled components.
+2. Go to the `src\GraphODataTemplateWriter\bin\debug` folder to find all compiled components.
 3. In that folder, modify `.config\TemplateWriterSettings.json` to specify your template mapping see [Template Writer Settings](##Template-Writer-Settings) for more details.
-4. Open a command prompt as administrator in the same folder and run `Vipr.exe <path-or-url-to-metadata> --writer="GraphODataTemplateWriter"`
+4. Open a command prompt as administrator in the same folder and run `Vipr.exe <path-or-url-to-metadata> --writer="GraphODataTemplateWriter"` where the supplied path refers to the [metadata file](graphmetadata2.xml) in the root of this project.
```

Comment:
I might avoid this level of specificity, and instead favor something indicating that an example `$metadata` can be found at that path. Ideally, the codebase's maturity trends towards a point at which any arbitrary and valid OData 4 CSDL can be used. Just a thought 👍